vllm/model_executor/models/llama.py

```diff
@@ -52,6 +53,23 @@
 
 from .interfaces import SupportsLoRA
 
+_GGUF_KEYS_MAPPING = {
+    "token_embd": "model.embed_tokens",
+    "blk": "model.layers",
+    "ffn_up": "mlp.up_proj",
+    "ffn_down": "mlp.down_proj",
+    "ffn_gate": "mlp.gate_proj",
+    "ffn_norm": "post_attention_layernorm",
+    "attn_norm": "input_layernorm",
+    "attn_q": "self_attn.q_proj",
+    "attn_v": "self_attn.v_proj",
+    "attn_k": "self_attn.k_proj",
+    "attn_output": "self_attn.o_proj",
+    "output.qweight": "lm_head.qweight",
+    "output.qweight_type": "lm_head.qweight_type",
```

Comment:
Why is there a special qweight_type for the lm_head?

Reply:
This is unused anymore, we could remove this.